daemon/container_operations_unix.go

```diff
@@ -622,28 +652,14 @@ func (daemon *Daemon) ConnectToNetwork(container *container.Container, idOrName
 }
 
 func (daemon *Daemon) connectToNetwork(container *container.Container, idOrName string, updateSettings bool) (err error) {
-	if container.HostConfig.NetworkMode.IsContainer() {
-		return runconfig.ErrConflictSharedNetwork
-	}
-
-	if containertypes.NetworkMode(idOrName).IsBridge() &&
-		daemon.configStore.DisableBridge {
-		container.Config.NetworkDisabled = true
-		return nil
-	}
-
-	controller := daemon.netController
-
-	n, err := daemon.FindNetwork(idOrName)
+	n, err := daemon.updateNetworks(container, idOrName, updateSettings)
```

Comment:
@cpuguy83 I guess @coolljt0725 must ignore this and https://github.com/docker/docker/pull/18906/files#r49092752. Pls confirm.